compiler/vm.nim

```diff
@@ -2077,6 +2081,11 @@ proc execute(c: PCtx, start: int): PNode =
   newSeq(tos.slots, c.prc.maxSlots)
   result = rawExecute(c, start, tos).regToNode
 
+proc execFromCtx*(c: PCtx, pc = -1, tos: PStackFrame = nil): PNode =
```

Comment:
I wanted to make this a proc called `resume` in `nimeval`, but `rawExecute` isn't exported. Should I export `rawExecute`, and get rid of this? That would also allow `Interpreter.graph` to stay private.